doc/tutorials/01_Param.ipynb

```diff
@@ -0,0 +1,338 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b98680e2-3c2c-461a-aa7e-ff383cf55e2d",
+   "metadata": {},
+   "source": [
+    "<style>div.container { width: 100% }</style>\n",
+    "<img style=\"float:left;  vertical-align:text-bottom;\" height=\"65\" width=\"172\" src=\"../_static/logo_stacked.png\" />\n",
+    "<div style=\"float:right; vertical-align:text-bottom;\"><h2>Tutorial 1. Param</h2></div>\n",
+    "<br><br>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "05081193-c6b8-46c9-82a8-3057983b6ef7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import param\n",
+    "import panel as pn\n",
+    "\n",
+    "pn.extension('tabulator')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fdfc0ac9-b516-49b6-a21b-e173b5d2de11",
+   "metadata": {},
+   "source": [
+    "# Param Basics for Panel\n",
+    "\n",
+    "Panel and other projects in the HoloViz ecosystem all build on Param. Param provides a framework to add validation, documentation and interactivity to a project. It is similar to more modern projects such as Pydantic but focuses primarily on providing APIs that make it easy to express complex dependencies and UI interactions.\n",
+    "\n",
+    "Here we won't focus on how param works but instead try to get you to understand the fundamentals that will allow you to effectively use Panel, express interactivity and reactivity in idiomatic ways and lastly how to structure your code to make it easily reusable and avoid callback hell."
```

Comment:
It seems to me this is going a little too fast over basics Param. If this guide is intended to build complex apps, we'll want to nudge people into writing more Parameterized classes. This guide doesn't really explain how as it currently is.